enhancements/network/ovn-k-localnet-api.md

```diff
@@ -0,0 +1,516 @@
+---
+title: user-defined-network-segmentation
+authors:
+  - "@ormergi"
+reviewers:
+  - "@maiqueb"
+  - TBD
```

Comment:
@npinaeva you're my go-to API person in the SDN team. 

Can we beg you for some reviews ?... Or has tried to keep this really short.